### PR TITLE
Implement action to test and merge green head

### PR DESCRIPTION
**Pull request by berwinlan**
DO NOT MERGE PRs TO TRUE-HEAD IF THEY DON'T HAVE TESTS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "noodlemaps",
+  "version": "1.0.0",
+  "description": "Better multi-stop path planning built on google maps",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (hnvakil) [inline package.json:7]: Is this "error" intentional?

**Comment by berwinlan** [comment]
This needs to be merged into `true-head` to work, since we're watching for pushes on true-head. To the best of my knowledge this should work

**Review by benmor20 — APPROVED** [review]
Looks good to me

**Comment by hnvakil** [comment]
looks good to me too

**Review by hnvakil — APPROVED** [review]
still looks good

**Pull request merged**